app/controllers/prior_authority/steps/check_answers_controller.rb

```diff
@@ -1,16 +1,25 @@
-# TODO: This is a placeholder to show how the before_action would be implemented
-# expect to repeat this before form submission as well.
-#
 module PriorAuthority
   module Steps
-    class CheckAnswersController < Nsm::Steps::BaseController
+    class CheckAnswersController < PriorAuthority::Steps::BaseController
+      # TODO: This is a placeholder to show how the before_action would be implemented
+      # expect to repeat this before form submission as well.
+      #
       # before_action :check_complete?
```

Comment:
Since we have this available, we might as well uncomment this and start taking advantage of it.

Reply:
This was added from david via a rebase and not sure what it is intended to do or the logic of it.

Reply:
So this is the last-ditch saviour if I open a 'Change' link in a new tab from the Check Answers screen then change an answer in such a way that it puts the application into an invalid state (e.g change service type but not update service costs) and then go back to the original tab and click 'Submit'  - the TaskList should be the single source of truth about whether the application is in a submittable state, so this lets us check in with it.

Reply:
cool, just spoke to Dave, I am going to work on this in the sub task [1132](https://dsdmoj.atlassian.net/jira/software/c/projects/CRM457/boards/1192?selectedIssue=CRM457-1132) for the CYA confirmation form which i am working on now.